fml/message_loop_impl.cc

```diff
@@ -119,17 +119,22 @@ void MessageLoopImpl::DoTerminate() {
 
 void MessageLoopImpl::FlushTasks(FlushType type) {
```

Comment:
are there any unit tests for `FlushTasks`?

Reply:
`FlushTasks` is private so there weren't unit tests for it.

Reply:
what about the public API `RunExpiredTasksNow` or `RunSingleExpiredTaskNow`?  I think we can repro the bug by just calling those, no?

Reply:
Yes you are right. `FollowingTasksSwitchQueueIfFirstTaskMergesThreads` also catches the bug but testing `RunExpiredTasksNow` would be better. I will see to set it up.

Reply:
done